### PR TITLE
Dont FFDC managed object exceptions

### PR DESCRIPTION
**Pull request by benjamin-confino**
Part of issue #3876

### CODE REVIEW

**Comment by benjamin-confino** [comment]
#build

File: dev/com.ibm.ws.ejbcontainer.core/src/com/ibm/ejs/container/ManagedBeanOBase.java [hunk 1]
```diff
@@ -147,10 +147,18 @@ protected void injectInstance(ManagedObject<?> managedObject, Object instance, I
...
-                    Tr.exit(tc, "injectInstance : Injection failure", t);
-                throw ExceptionUtil.EJBException("Injection failure", t);
+                if (! (t instanceof ManagedObjectException)) {
+                    FFDCFilter.processException(t, CLASS_NAME + ".injectInstance", "134", this);
+                    if (isTraceOn && tc.isEntryEnabled())
+                        Tr.exit(tc, "injectInstance : Injection failure", t);
+                    throw ExceptionUtil.EJBException("Injection failure", t);
+                } else {
+                    Throwable cause = t.getCause();
+                    FFDCFilter.processException(cause, CLASS_NAME + ".injectInstance", "134", this);
```
Comment (tkburroughs) [inline dev/com.ibm.ws.ejbcontainer.core/src/com/ibm/ejs/container/ManagedBeanOBase.java]: technically, ever FFDC should have a different probe ID (usually line number), I'd change this one to "157"

File: dev/com.ibm.ws.ejbcontainer.core/src/com/ibm/ejs/container/ManagedBeanOBase.java [hunk 2]
```diff
@@ -366,13 +373,24 @@ protected InvocationContextImpl createInvocationContext() {
...
+                FFDCFilter.processException(t, CLASS_NAME + ".callAroundConstructInterceptors", "240", this);
+                // If a root cause exception was captured, then return the context and let the caller
+                // properly report the failure and cause; otherwise throw generic EJBException.
+                if (invCtx.ivAroundConstructException != null) {
+                    return invCtx;
+                }
+                throw ExceptionUtil.EJBException("AroundConstruct interceptor failure", t);
+            } else {
+                Throwable cause = t.getCause();
+                FFDCFilter.processException(cause, CLASS_NAME + ".callAroundConstructInterceptors", "240", this);
```
Comment (tkburroughs) [inline dev/com.ibm.ws.ejbcontainer.core/src/com/ibm/ejs/container/ManagedBeanOBase.java]: as above, change probe ID to a different value

**Comment by benjamin-confino** [comment]
#build

**Comment by benjamin-confino** [comment]
And here's a combined build that reverts the allow ffdcs

https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_vizH4GtYEeifxOsxEpHfDQ&tab=com.ibm.team.build.web.ui.internal.editors.result.properties.PropertiesProvider

**Pull request merged**